incent-config/Dangerfile

```diff
@@ -132,6 +133,10 @@ def branch_contains_prefix?
   prefixes.any? { |prefix| branch_name.start_with?(prefix) }
 end
 
+def branch_name_contains_underscore?
+  return true if branch_name.include? "_"
```

Comment:
The "idiomatic Ruby" version of this line would be:

```rb
branch_name.include?('_')
```

* no need for `return` on last line
* no need for `true if` when that line is already returning a boolean
* we don't have RuboCop here but we prefer single over double quotes
* we don't have RuboCop here but we prefer parenthesis around method arguments